platform/external-system-impl/src/com/intellij/openapi/externalSystem/service/settings/AbstractExternalSystemConfigurable.java

```diff
@@ -81,9 +81,8 @@ public JComponent createComponent() {
       SystemSettings settings = getSettings();
       prepareSystemSettings(settings);
       prepareProjectSettings(settings);
-      return myComponent;
     }
-    return myComponent;
+    return new JBScrollPane(myComponent, ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED, ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);
```

Comment:
This scroll pane has a border by default. Could you remove them from the pane?

## Actual
<img width="1187" alt="Screenshot 2024-10-25 at 16 32 21" src="https://github.com/user-attachments/assets/b0b2427b-5a09-49ee-849c-0bd7dbac7818">

## Expected
![image](https://github.com/user-attachments/assets/1d1f548c-4abb-46ae-aeec-b71442cd5469)